docs/commands/SetTime2000.md

```diff
@@ -0,0 +1,84 @@
+# SetTime2000
+
+Time correction command.
+It is used when the time difference is more than `127` seconds.
+
+
+## Request
+
+### Format
+
+>  Size | Type     | Field
+> ------|----------|-------
+>  1    | byte     | command id = `0x02`
+>  1    | byte     | command size = `5`
+>  1    | byte     | [sequence number](#sequence-number)
+>  4    | int32_be | [seconds](#seconds)
+
+### Parameters
+
+#### **sequence number**
+
+It's a unique number for each time correction request.
+Use the last received [sequence number](./uplink/Time2000.md#sequence-number) + `1` as a starting point.
+Then increment this value for each following request.
```

Comment:
Increment is not necessary - it can be a pseudo-random value, just need to be unique?